lib/aquilon/worker/templates/city.py

```diff
@@ -30,7 +30,7 @@ class PlenaryCity(Plenary):
 
     @classmethod
     def template_name(cls, dbcity):
-        return "%s/%s/%s/config" % (cls.prefix, dbcity.hub.fullname.lower(),
+        return "%s/%s/%s/config" % (cls.prefix, dbcity.continent.name,
```

Comment:
we cannot yet merge this cause we need to fix a 'bug' in city templates for timezones. but in general we agree with the change - just let us prepare for it.

Reply:
No problem.